File: 1324C.java

import java.util.Scanner;

public final class Scratch
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        int t = scanner.nextInt();
        for (int i = 0; i < t; ++i)
            solve(scanner.next());
    }

    private static void solve(String test)
    {
        int pos = test.length();
        int d = 1;
        while (pos >= 0)
        {
            int jump = 1;
            while (pos - jump >= 0 && test.charAt(pos - jump) != 'R') jump += 1;
            d = Math.max(jump, d);
            pos -= jump;
        }
        System.out.println(d);
    }
}

File: 1324C.py
# http://codeforces.com/problemset/problem/1324/C
t = int(input())
test_cases = [input() for i in range(t)]


def solve_test(test_case):
    pos = len(test_case)
    d = 1
    while pos >= 0:
        jump = 1
        while test_case[pos - jump] != 'R':
            jump += 1
        d = max(jump, d)
        pos -= jump
    return d


def solve(test_cases):
    for test in test_cases:
        print(solve_test(test))


solve(test_cases)
